client/web/src/components/views/readonly-client-view.tsx

```diff
@@ -23,6 +23,25 @@ export default function ReadonlyClientView({
   auth?: AuthResponse
   sessions: SessionsCallbacks
 }) {
+  const newSession = async () => {
+    return sessions
+      .new()
+      .then((url) => {
+        if (url) {
+          window.open(url, "_blank")
+          sessions.wait()
+        }
+      })
+      .catch((e) => console.error(e))
+  }
```

Comment:
You know what, we could just have the `useAuth` hook just export this function rather than returning `sessions` with the two session functions. Maybe a little cleaner over there than here.

Reply:
done